alert-common/src/main/java/com/synopsys/integration/alert/common/action/CustomFunctionAction.java

```diff
@@ -22,28 +22,53 @@
  */
 package com.synopsys.integration.alert.common.action;
 
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
 import org.springframework.http.HttpStatus;
 import org.springframework.web.server.ResponseStatusException;
 
+import com.github.jsonldjava.shaded.com.google.common.base.Predicate;
```

Comment:
I think you want to use Java's Predicate and not this Predicate object.

Reply:
Wow, good catch. Why is this even a suggestion from Intellij haha?